org.gazebosim.Gazebo.metainfo.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>org.openchemistry.Avogadro2</id>
```

Comment:
this doesn't match the app-id of the app you're installing.

Reply:
Thanks, it is just place holder for now, will change it soon.

Reply:
it makes the build fail